src/store/notifications.js

```diff
@@ -54,7 +54,7 @@ function _addNotification(notification) {
  */
 export const notify = (notification) => (dispatch, getState) => {
   const notifications = getState().notifications
-  const doesNotifExist = notifications.find(notif => notif.id === notification.id)
+  const doesNotifExist = notifications.some(notif => notif.id === notification.id)
```

Comment:
`Array#some` has much wider support (it's part of ES5), and its semantics are better applicable to this case: it returns `true` if the predicate is truthy for some of the elements and `false` otherwise